srcpkgs/libclc/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'libclc'
+pkgname=libclc
+version=11.0.0
+revision=1
+wrksrc="${pkgname}-${version}.src"
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release"
+hostmakedepends="cmake clang llvm python3 libedit-devel libffi-devel
+ ncurses-devel zlib-devel"
+short_desc="Open implementation of the OpenCL C programming language"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="BSD-3-Clause, MIT"
+homepage="https://libclc.llvm.org"
+distfiles="https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/libclc-${version}.src.tar.xz"
+checksum=5a0a2f67b85ede09b2f08c071c5b0b6b5ad56bfb00b39a22c369afcc613c42ee
+replaces="libclc-git>=0"
+
+do_configure() {
+	# We really do a native build.
+	# The outcome is just header files.
+	unset CC CXX LD CFLAGS CXXFLAGS LDFLAGS
+	mkdir -p build
+	cd build
+	cmake ${configure_args} ..
```

Comment:
Why not the cmake build style?

Reply:
Because it tries to cross compile for the target architecture.

Reply:
Doesn't explain why bother with the do_build or do_install steps manually

Reply:
True. I didn't think about overriding just the `do_configure()` step. I will change that.